docs/api-reference/data-fetching/getServerSideProps.md

```diff
@@ -0,0 +1,111 @@
+---
+description: Fetch data on each request with `getServerSideProps`. Learn more about this API for data fetching in Next.js.
+---
+
+# `getServerSideProps`
+
+<details>
+  <summary><b>Version History</b></summary>
+
+| Version   | Changes                                                             |
+| --------- | ------------------------------------------------------------------- |
+| `v10.0.0` | `locale`, `locales`, `defaultLocale`, and `notFound` options added. |
+| `v9.3.0`  | `getServerSideProps` introduced.                                    |
+
+</details>
+
+When exporting an `async` function called `getServerSideProps` (server-side rendering) from a page, Next.js will pre-render this page on each request using the data returned by `getServerSideProps`.
+
+```js
+export async function getServerSideProps(context) {
+  return {
+    props: {}, // will be passed to the page component as props
```

Comment:
Is all of this content copy/paste or did you change parts? Just trying to understand how much to read into it

Reply:
Copy paste then some copy change and new sections. Its mostly the same content, I've gone through it though